source/Octopus.Client/OctopusClientFactory.cs

```diff
@@ -8,6 +8,13 @@ namespace Octopus.Client
     /// </summary>
     public class OctopusClientFactory : IOctopusClientFactory
     {
+        private static string requestingTool;
```

Comment:
Not enthused by static methods like this. It'd be better to pass it in in `CliProgram` when configuring the builder.